checks/postgres/check.go

```diff
@@ -0,0 +1,126 @@
+package postgres
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/jmoiron/sqlx"
```

Comment:
Let's use `github.com/lib/pq` only, having `github.com/jmoiron/sqlx` is too much for this library

Reply:
And If possible having only standard lib for this would be even better

Reply:
Agree, will remove `logrus` usage as well in favour of the `log` and callback to override this.